blog/_posts/2022-01-19-learn-scala.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Vincent Derouand, and Valérie Pedroni (Scala Center)
+title: "Learn Scala"
+---
+
+Happy new year 2022! The beginning of the year is a good time to refresh 
+your programming skills. We want to use this opportunity to report on what 
+happened on the MOOCs in 2021, and to share our plans for 2022.
+
```

Comment:
```suggestion
Happy new year to everyone! We hope 2022 will be of great joy to everyone and will be a source of great collaboration between the Scala Center and the Scala community!
As you might be listing your 2022 goals why not include Scala in it? Refresh your programming skills? Acquire new knowledge and get a new certificate? Take a look at our online courses designed for both beginners and programmers to use Scala language.
```

Reply:
Thank you for the suggestion! I reworked the introduction in 430b3bf0, let me know what you think.